mungegithub/mungegithub.go

```diff
@@ -105,7 +105,7 @@ func main() {
 			}
 			errInit := mungers.InitializeMungers(&config.Config, &config.Features)
 			if errInit != nil {
```

Comment:
Please use the pattern:

```
if err := mungers.InitializeMungers(&config.Config, &config.Features); err != nil {
```

Here and above on line 99. This way you can use the same variable name while avoiding this type of bug (err is only declared in the body of the if).